src/BSON/Binary.c

```diff
@@ -377,32 +431,6 @@ static int php_phongo_binary_compare_objects(zval* o1, zval* o2) /* {{{ */
 	return zend_binary_strcmp(intern1->data, intern1->data_len, intern2->data, intern2->data_len);
 } /* }}} */
 
-static HashTable* php_phongo_binary_get_properties_hash(phongo_compat_object_handler_type* object, bool is_debug) /* {{{ */
```

Comment:
Various `get_properties_hash` functions needed to be moved earlier in the file because they're used by `__serialize` methods. The alternative was to just add forward declarations at the top of the file, but that will ultimately be sorted by [PHPC-1728](https://jira.mongodb.org/browse/PHPC-1728).